src/NuGet.Clients/NuGet.VisualStudio/NuGet.VisualStudio.csproj

```diff
@@ -9,6 +9,9 @@
     <Shipping>true</Shipping>
     <PackProject>true</PackProject>
     <IncludeInVSIX>true</IncludeInVSIX>
+    <!-- VS Extensibility APIs should use package version corresponding to VS version it targets, to reduce confusion. Also keep assembly version stable to minimize binding redirect issues. -->
+    <PackageVersion>$(NuGetSdkVsSemanticVersion)$(PreReleaseInformationVersion)</PackageVersion>
+    <AssemblyVersion>$(NuGetSdkVsSemanticVersion).0</AssemblyVersion>
```

Comment:
This is going to change the package and assembly version scheme, which we should not do in a patch version of 6.0